swatch-billable-usage/src/main/java/com/redhat/swatch/billable/usage/services/BillableUsageService.java

```diff
@@ -201,6 +213,27 @@ private BillableUsageCalculation calculateBillableUsage(
         .build();
   }
 
+  private void createGratisRemittance(BillableUsage usage) {
```

Comment:
Rather than repeat a bunch of logic from `createRemittance`, it'd be better to add a parameter to createRemittance for the `RemittanceStatus value`.

We _do_ want the value to be populated using the same logic:
```java
    // Remitted value should be set to usages metric_value rather than billing_value
    newRemittance.setRemittedPendingValue(usageCalc.getRemittedValue());
    newRemittance.setRemittancePendingDate(usageCalc.getRemittanceDate());
```.

So the only difference should be in the `status` value.